package.json

```diff
@@ -36,7 +36,7 @@
   "homepage": "https://github.com/lifion/lifion-kinesis#readme",
   "main": "lib/index.js",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=17.0.1 <16.10.0 >=10.0.0"
```

Comment:
In the ticket that you create, please mention that this needs to be updated too.

Reply:
Will do, I'll add a short description of the problem with Node 16, and link the issue @node.